packages/listview/src/components/DownloadModal/index.d.ts

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable @typescript-eslint/ban-types */
+import { CSSProperties } from 'react';
+
+type Max = string | number;
+
+export interface DownloadModalProps {
+    /** The title at the top of the DownloadModal component. */
+    title?: string;
+    /** The description in the DownloadModal component. */
+    description?: string;
```

Comment:
title and description must be a ReactNode, they can pass a string but they can also pass whatever they want with its own styles or a react intl FormattedMessage component for example